src/librustdoc/html/layout.rs

```diff
@@ -74,10 +74,12 @@ r##"<!DOCTYPE html>
         {sidebar}
     </nav>
 
-    <button id="theme-picker">
-        <img src="{root_path}brush.svg" width="18" alt="Pick another theme!">
+    <div class="theme-picker">
```

Comment:
do we know *why* this change makes it work? semantically, this should be a `<button>`.

Reply:
Now it's a `<button>` inside this `<div>` so i would assume that there's some layout trickery going on.

Reply:
Just like @QuietMisdreavus said. :) Before it was buttons inside a button, now it's buttons inside a div. More logical for everyone (browsers included).

Reply:
Oh right, i was missing the other `<button>`s being put inside this one. I like this version more.